cmd/ocsp-responder/main.go

```diff
@@ -106,25 +132,18 @@ func main() {
 
 		auditlogger.Info(app.VersionString())
 
-		// Configure DB
-		dbMap, err := sa.NewDbMap(c.OCSPResponder.DBConnect)
-		cmd.FailOnError(err, "Could not connect to database")
-		sa.SetSQLDebug(dbMap, c.SQL.SQLDebug)
-
-		// Load the CA's key so we can store its SubjectKey in the DB
-		caCertDER, err := cmd.LoadCert(c.Common.IssuerCert)
-		cmd.FailOnError(err, fmt.Sprintf("Couldn't read issuer cert [%s]", c.Common.IssuerCert))
-		caCert, err := x509.ParseCertificate(caCertDER)
-		cmd.FailOnError(err, fmt.Sprintf("Couldn't parse cert read from [%s]", c.Common.IssuerCert))
-		if len(caCert.SubjectKeyId) == 0 {
-			cmd.FailOnError(fmt.Errorf("Empty subjectKeyID"), "Unable to use CA certificate")
+		config := c.OCSPResponder
+		var source cfocsp.Source
+		if config.Source.DBConnect != "" {
```

Comment:
I'm not super stoked about this union style config but I don't immediately have a better option. (Well, there's the "one url with file:// or mysql+tcp:// scheme to make the right thing happen)

But in the union case, we need a check for both and neither being specified to raise an error.